passage/src/androidTest/java/id/passage/android/IntegrationTestConfig.kt

```diff
@@ -9,5 +12,20 @@ internal class IntegrationTestConfig {
         const val EXISTING_USER_EMAIL_OTP = "authentigator+1716916054778@ncor7c1m.mailosaur.net"
         const val EXISTING_USER_EMAIL_MAGIC_LINK = "authentigator+1716572384858@ncor7c1m.mailosaur.net"
         const val DEACTIVATED_USER_EMAIL_MAGIC_LINK = "authentigator+1716778790434@ncor7c1m.mailosaur.net"
+        const val AUTH_TOEKN = BuildConfig.OTP_TEST_USER_AUTH_TOKEN
+        val CURRENT_USER =
+            PassageUser.convertToPassageUser(
+                User(
+                    email = "authentigator+1716916054778@ncor7c1m.mailosaur.net",
+                    emailVerified = true,
+                    id = "Cf70Zkwn1ywnpuua1ZFF717G",
+                    phone = "",
+                    phoneVerified = false,
+                    status = UserStatus.active,
+                    userMetadata = null,
+                    webauthn = false,
+                    webauthnTypes = emptyList(),
+                ),
+            )
```

Comment:
I believe you should be able to init the PassageUser instance directly? Maybe like:
```suggestion
            PassageUser(
                  email = "authentigator+1716916054778@ncor7c1m.mailosaur.net",
                  emailVerified = true,
                  id = "Cf70Zkwn1ywnpuua1ZFF717G",
                  phone = "",
                  phoneVerified = false,
                  status = UserStatus.active,
                  userMetadata = null,
                  webauthn = false,
                  webauthnTypes = emptyList(),
            )
```

Reply:
Since 'PassageUser' has private constructor, it's not possible